client/src/components/common/PlayerList.jsx

```diff
@@ -4,7 +4,9 @@ import PlayerListEntry from './PlayerListEntry.jsx';
 
 const PlayerList = (props) => (
   <div className="player-list">
-    {props.players.map((player, k) => <PlayerListEntry player={player} key={k} /> )}
+  
+    {props.players.map((player, k) => <div><PlayerListEntry player={player} key={k} /> </div>)}
```

Comment:
The wrapper div here looks redundant at first glance. Is there a reason why this is necessary? The PlayerListEntry actually already has a div wrapper inside of it.

Reply:
good call.  It was leftover from an earlier version.  I've removed it

Reply:
Okay, go ahead and push that change up so it's updated in the pull request.